### ISSUE TITLE
Core Person Vocabulary 2.00 : countryOfBirth described as residency

### ISSUE THREAD

**Issue by bouil** [SEMICeu/Core-Person-Vocabulary#42]
Hi,

It looks like there is a mistake in the core-person-ap.ttl and core-person-SHACL.ttl file. 
The countryOfBirth is described as the residency.
In the SHACL, the countryOfBirth is present twice. 

```
<http://www.w3.org/ns/person#countryOfBirth> a rdf:Property;
  rdfs:label "residency"@en,
    "country of birth"@en .
```

https://github.com/SEMICeu/Core-Person-Vocabulary/blob/20203b0312397ccbfc680abd71129c94a7f37863/releases/2.00/voc/core-person-ap.ttl#L114

```
    shacl:class dc:Jurisdiction;
    shacl:description "Jurisdiction where the Person has their dwelling."@en;
    shacl:name "residency"@en;
    shacl:path <http://www.w3.org/ns/person#countryOfBirth>
```
https://github.com/SEMICeu/Core-Person-Vocabulary/blob/20203b0312397ccbfc680abd71129c94a7f37863/releases/2.00/shacl/core-person-SHACL.ttl#L242

```
    shacl:class dc:Location;
    shacl:description "The country in which the Person was born."@en;
    shacl:name "country of birth"@en;
    shacl:path <http://www.w3.org/ns/person#countryOfBirth>
```

https://github.com/SEMICeu/Core-Person-Vocabulary/blob/20203b0312397ccbfc680abd71129c94a7f37863/releases/2.00/shacl/core-person-SHACL.ttl#L172

**Comment by EmidioStani** [comment]
Hello @bouil thanks for reporting this. It will be part of the next release.

**Comment by EmidioStani** [comment]
This issue is solved in the release [2.1.0](https://semiceu.github.io/Core-Person-Vocabulary/releases/2.1.0/)

If you spot similar issues do not hesitate to report it, thanks !
